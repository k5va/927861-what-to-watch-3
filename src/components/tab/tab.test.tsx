import * as renderer from "react-test-renderer";
import Tab from "./tab";

it(`Tab should render correctly`, () => {
  const wrapper = renderer
    .create(
        <Tab name="Hello">
          <h1>Hello world!</h1>
        </Tab>
    )
    .toJSON();
  expect(wrapper).toMatchSnapshot();
});
