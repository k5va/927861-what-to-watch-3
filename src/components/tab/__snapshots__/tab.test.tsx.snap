// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tab should render correctly 1`] = `
<h1>
  Hello world!
</h1>
`;
